NBitcoin/BitcoinCore/Coins.cs

```diff
@@ -88,58 +63,43 @@ public Coins(Transaction tx, int height)
 
 		private void UpdateValue()
 		{
-			_Value = vout.Where(o => !IsNull(o))
+			Value = Outputs.Where(o => !IsNull(o))
 							.Select(o => o.Value).Sum();
 		}
 
-		private bool IsNull(TxOut o)
-		{
-			return o.Value.Satoshi == -1;
-		}
-
-		public bool IsEmpty
-		{
-			get
-			{
-				return vout.Count == 0;
-			}
-		}
+		private bool IsNull(TxOut o) => o.Value.Satoshi == -1;
+		public bool IsEmpty => Outputs.Count == 0;
 
 		private void Cleanup()
 		{
-			var count = vout.Count;
+			var count = Outputs.Count;
 			// remove spent outputs at the end of vout
 			for(int i = count - 1; i >= 0; i--)
 			{
-				if(IsNull(vout[i]))
-					vout.RemoveAt(i);
+				if(IsNull(Outputs[i]))
+					Outputs.RemoveAt(i);
 				else
 					break;
 			}
 		}
 
-		public int UnspentCount
-		{
-			get
-			{
-				return vout.Where(c => !IsNull(c)).Count();
-			}
-		}
+		public int UnspentCount => Outputs.Where(c => !IsNull(c)).Count();

```

Comment:
I think that could be more easy to read if you change ``Outputs.Where(predicate).Sum()`` by ``Outputs.Sum(predicate)``